fw/http_limits.c

```diff
@@ -1099,6 +1131,12 @@ frang_http_req_process(FrangAcc *ra, TfwConn *conn, TfwFsmData *data,
 		{
 			T_FSM_EXIT();
 		}
+		/* Optional protection against domain fronting attacks */
+		if (!f_cfg->http_allow_domain_fronting
```

Comment:
Why do we need a new configuration option for this? I think we should just combine it with `http_strict_host_checking`. Especially that both the options are in permissive mode by default

Reply:
These options are not related: http_strict_host_checking comes from "SHOULD" terms of HTTP RFCs. http_allow_domain_fronting that originates from the TLS world.

After all, I believe the more flexible tempesta is, the more use cases it may fit.

For example, domain fronting would make `cache.test_cache.TestCacheVhost` test completely useless.

Reply:
Merged these options together